code/server/server.cpp

```diff
@@ -0,0 +1,178 @@
+#include <iostream>
+#include <yarp/os/all.h>
+
+using namespace std;
+using namespace yarp::os;
+
+#define FIFO_SIZE 10
+
+volatile int intFIFOtmp[FIFO_SIZE];
+volatile int intFIFOnum;
+
```

Comment:
I couldn't implement it in good way...
But, I now fixed it to read the FIFO directly.